src/Payum/Paypal/ExpressCheckout/Nvp/Action/Api/DoVoidAction.php

```diff
@@ -0,0 +1,61 @@
+<?php
+namespace Payum\Paypal\ExpressCheckout\Nvp\Action\Api;
+
+use Payum\Core\Bridge\Spl\ArrayObject;
+use Payum\Core\Exception\RequestNotSupportedException;
+use Payum\Core\GatewayAwareInterface;
+use Payum\Core\GatewayInterface;
+use Payum\Paypal\ExpressCheckout\Nvp\Request\Api\DoVoid;
+use Payum\Paypal\ExpressCheckout\Nvp\Request\Api\GetTransactionDetails;
+
+class DoVoidAction extends BaseApiAwareAction implements GatewayAwareInterface
```

Comment:
Maybe so, but I duplicated the [DoCaptureAction](https://github.com/Payum/Payum/blob/master/src/Payum/Paypal/ExpressCheckout/Nvp/Action/Api/DoCaptureAction.php) for consistency.  If there are performance or implementation concerns, wouldn't that be better as a separate issue?